src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
```

Comment:
Link to a SCSS file should be replaced with a compiled CSS file. Browsers do not understand SCSS syntax, so ensure you are linking to a valid `.css` file after compilation.